include/links-directory.php

```diff
@@ -285,4 +277,38 @@ public function rewrite_rules( $rules ) {
 
 		return $newrules;
 	}
+
+	/**
+	 * Removes hooks to filter rewrite rules, called when switching blog @see {PLL_Base::switch_blog()}.
+	 * See `self::prepare_rewrite_rules()` for added hooks.
+	 *
+	 * @since 3.5
+	 *
+	 * @return void
+	 */
+	public function remove_filters() {
+		parent::remove_filters();
+
+		remove_all_actions( 'pll_prepare_rewrite_rules' );
```

Comment:
Something has been lost in the loop.
This part is already done in the parent method.

Reply:
Fixed in aa8be6e7.